spec/models/smile_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+describe Smile, type: :model do
+  it { is_expected.to belong_to :user }
+  it { is_expected.to belong_to :photo }
```

Comment:
There are more tests too which would be helpful here (i.e. checking that unauthenticated users cannot add smiles)